### DISCUSSION TITLE
Color styles with two or more fills not supported as tokens

### DISCUSSION THREAD

**Discussion by ianturk** [six7/figma-tokens#712] [Ideas]
We are exploring some styles for backplates that use two fills, a solid backplate with an overlay of transparent white or black. These styles aren't recognized when importing styles to Figma Tokens. Would love for Figma Tokens to support a token with multiple fills.

**Comment by six7** [comment]
This issue should be about what you described, right? https://github.com/six7/figma-tokens/issues/152

I'm still on the fence as I dont't yet know how this would translate to code. I feel the better solution would be a way to do a colormix with a modifier, a concept that would work for many different transformations (alpha, math, color lighten darken mix)
